scripts/openapi-pydantic-models-generator.bash

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#!/bin/bash
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -o errexit
+set -o nounset
+set -o pipefail
+IFS=$'\n\t'
+
+PYTHON_VERSION=3.9.12
+IMAGE_NAME="local/datamodel-code-generator:${PYTHON_VERSION}"
+WORKDIR="$(pwd)"
+
+
+Build()
+{
+  docker buildx build \
+    --build-arg PYTHON_VERSION="${PYTHON_VERSION}" \
+    --build-arg HOME_DIR="/home/$USER" \
+    --tag "$IMAGE_NAME" \
+    - <<EOF
+FROM python:${PYTHON_VERSION}-slim
+RUN pip install datamodel-code-generator[http]
```

Comment:
maybe consider pinning this library as well as the docker-compose API version/spec version.

Reply:
it's a tool, I think we do not pin them. but in case this starts breaking we can.